fluent-plugin-logdna.gemspec

```diff
@@ -1,23 +1,24 @@
-# -*- encoding: utf-8 -*-
-require 'date'
+# frozen_string_literal: true
+
+require "date"
 
 Gem::Specification.new do |s|
-  s.name        = 'fluent-plugin-logdna'
-  s.version     = '0.3.1'
+  s.name        = "fluent-plugin-logdna"
+  s.version     = "0.3.1"
```

Comment:
If we're publishing through CircleCI, should this version be variable/auto-incrementing rather than hardcoded?

Reply:
No, it's always been the case for all we have using CircleCI that we have been updating the version manually